content/courses/Functions-and-Scope.md

```diff
@@ -11,14 +11,14 @@ tags:
   - "Context"
   - "Closures"
 description: "If you want to learn about JavaScript functions, then this is the course for you. Did you know that everything is a function? No? How about binding functions to Objects? Closures? Learn the digestible rules of arrow functions and when to use them best. What the heck is a this anyway? Become familiar with scope and context in JavaScript and use them to your advantage. By the end of this course, you will become familiar with all of these topics and more."
-lessons: 
+lessons:
+  - link: "scope"
+    title: Scope
+    description: "Dive in head first into semantics of _scope_, compare and contrast it with context and build the confidence you need to understand one of the most fundamental concepts in JavaScript."
```

Comment:
`...compare and contrast it with context and build` => Missing a comma after `with context`